WPMedia/feed-components
# THEMES-1994: ANS block custom field

The ANS block was missing the image height and width custom field. This change adds it in.

**Testing Instructions**

- Checkout branch `git checkout THEMES-1994`
- Start local OBF feature pack with `npx fusion start -f -l @wpmedia/ans-feature-block`.
- Add the "ANS" block to a page or template, with the global content set to the default `feed-content-api` content source.
- Adjust the image width and height values on the block's custom field.
- With the output set to `json`, you should see that resizer v2 urls in the feed have the height and width specified in the custom field.
